pattern-library/aquent/components/forms/input/markup.html

```diff
@@ -1,8 +1,12 @@
+<p>Standard text input</p>
+<input type="text" placeholder="Name">
 <p>Select Element</p>
-<select>
-  <option value="" disabled="" hidden="" selected="">I am looking for</option>
-  <option value="Contact">I am looking for talent</option>
-  <option value="Talent">I am looking for work</option>
-</select>
+<div class="select-box">
+  <select>
+    <option value="" disabled="" hidden="" selected="">I am looking for</option>
```

Comment:
Suggestion: make these generic, like
"Default text" and "Option 1" and "Option 2". Basically, get rid of anything that makes it specific to how we use it.